OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -141,6 +141,10 @@ public static void NotifyBlocker(this Actor self, IEnumerable<CPos> positions)
 					return false;
 			}
 
+			// enemy unit or any other friendly unit is standing on it.
```

Comment:
If I were to write "Position is blocked by some actor" then I wouldn't have put comment at all. I wanted to emphasize on "friendly unit"